src/types/Newsroom.ts

```diff
@@ -94,9 +94,31 @@ export interface Newsroom extends NewsroomRef {
     email_branding: Newsroom.EmailBranding;
 
     is_privacy_portal_enabled: boolean;
+    /**
+     * @deprecated Please use `policies.privacy_policy.link` instead.
+     */
     custom_privacy_policy_link: string | null;
     custom_data_request_link: string | null;
 
+    policies: {
+        privacy_policy:
+            | {
+                  link: string;
+              }
+            | {
+                  content: string;
+                  origin: 'default' | 'custom';
+              };
```

Comment:
This could be extracted to a for example Policy or PolicySource type

Reply:
Ideal would be ContentOrigin, but it feels a bit dramatic for a very niche thing. I'll keep it as is for, and we'll address if this area grows.